src/main/resources/assets/fiveminsurvival/lang/zh_cn.json

```
@@ -0,0 +1,108 @@
+	"item.fiveminsurvival.chicken_soup":"鸡肉汤",
+	"item.fiveminsurvival.cereal":"谷物",
+	"item.fiveminsurvival.bowl_water":"一碗水",
+	"item.fiveminsurvival.bowl_milk":"一碗牛奶",
+	"item.fiveminsurvival.beef_stew":"炖牛肉",
+	"item.fiveminsurvival.mashed_potatoes":"土豆泥",
+	"item.fiveminsurvival.chocolate_ice_cream":"巧克力冰淇淋",
+	"item.fiveminsurvival.manure":"肥料",
+	"item.fiveminsurvival.flint_arrow":"燧石箭",
+	"item.fiveminsurvival.silver_arrow":"白银箭",
+	"item.fiveminsurvival.copper_fishing_rod":"铜钓鱼竿",
+	"item.fiveminsurvival.copper_arrow":"铜箭",
+	"item.fiveminsurvival.copper_war_hammer":"铜战锤",
+	"item.fiveminsurvival.copper_sword":"铜剑",
+	"item.fiveminsurvival.copper_shovel":"铜铲",
+	"item.fiveminsurvival.copper_shears":"铜剪",
+	"item.fiveminsurvival.copper_scythe":"铜镰刀",
+	"item.fiveminsurvival.copper_pickaxe":"铜镐",
+	"item.fiveminsurvival.copper_mattock":"铜鹤嘴锄",
+	"item.fiveminsurvival.copper_knife":"铜小刀",
+	"item.fiveminsurvival.copper_hoe":"铜锄头",
+	"item.fiveminsurvival.copper_hatchet":"铜斧",
+	"item.fiveminsurvival.silver_fishing_rod":"白银钓鱼竿",
+	"item.fiveminsurvival.silver_war_hammer":"白银战锤",
+	"item.fiveminsurvival.silver_sword":"白银剑",
+	"item.fiveminsurvival.silver_shovel":"白银铲",
+	"item.fiveminsurvival.silver_shears":"白银剪刀",
+	"item.fiveminsurvival.silver_scythe":"白银镰刀",
+	"item.fiveminsurvival.silver_pickaxe":"白银镐",
+	"item.fiveminsurvival.silver_mattock":"白银鹤嘴锄",
+	"item.fiveminsurvival.silver_knife":"白银小刀",
+	"item.fiveminsurvival.silver_hoe":"白锄头",
+	"item.fiveminsurvival.silver_hatchet":"白银斧头",
+	"item.fiveminsurvival.copper_ingot":"铜锭",
+	"item.fiveminsurvival.silver_ingot":"白银锭",
+	"item.fiveminsurvival.copper_nugget":"铜块",
+	"item.fiveminsurvival.silver_nugget":"白银块",
+	"item.fiveminsurvival.gold_arrow":"金箭",
+	"item.fiveminsurvival.salad":"沙拉",
+	"item.fiveminsurvival.down_feather":"羽绒",
+	"block.fiveminsurvival.unbaked_cake":"未烘烤的蛋糕",
+	"block.fiveminsurvival.copper_ore":"铜矿",
+	"block.fiveminsurvival.silver_ore":"白银矿石",
+	"block.fiveminsurvival.shining_gravel":"发光碎石",
+	"block.fiveminsurvival.shining_pea_gravel":"发光碎石粒",
+	"block.fiveminsurvival.flax":"亚麻",
+	"block.fiveminsurvival.flint_crafting_table": "火石工具台",
+	"block.fiveminsurvival.copper_crafting_table": "铜工具台",
+	"block.fiveminsurvival.iron_crafting_table": "铁工具台",
+	"block.fiveminsurvival.silver_crafting_table": "白银工具台",
+	"block.fiveminsurvival.gold_crafting_table": "黄金工具台",
+	"block.fiveminsurvival.mithril_crafting_table": "秘银工具台",
+	"block.fiveminsurvival.ancient_metal_crafting_table": "古代金属工具台",
+	"block.fiveminsurvival.obsidian_crafting_table": "黑曜石工具台",
+	"block.fiveminsurvival.pea_gravel": "碎石粒",
+	"block.fiveminsurvival.clay_oven": "粘土烤箱",
+	"block.fiveminsurvival.hardened_clay_oven": "赤陶烤箱",
+	"block.fiveminsurvival.sandstone_oven": "砂岩烤箱",
+	"block.fiveminsurvival.cobblestone_furnace": "鹅卵石熔炉",
+	"block.fiveminsurvival.obsidian_furnace": "黑曜石炉",
+	"block.fiveminsurvival.netherrack_furnace": "空灵炉",
+	"block.fiveminsurvival.cobweb_block": "蜘蛛网块",
```

|Feedback Priority|
|---|
|🧗‍♀️ Boulder / **Blocking**|

```suggestion
	"block.fiveminsurvival.cobweb_block": "蜘蛛网方块",
```